tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -340,7 +348,12 @@ static int spi_rx_every_4(struct device *dev, struct spi_config *spi_conf)
 	};
 	int ret;
 
-	LOG_INF("Start");
+	if (IS_ENABLED(CONFIG_SPI_STM32_DMA)) {
+		LOG_INF("Skip every 4");
```

Comment:
why?

Reply:
This test makes sens when transferring with Fifo. In case of DMA, the transfer is started for a defined length in a complete burst.

Reply:
ok